pxtblocks/blocklyloader.ts

```diff
@@ -1183,6 +1196,13 @@ namespace pxt.blocks {
                     pxt.log("searching for: " + searchFor);
                     updateUsedBlocks = false;
                     if (!blocks) return;
+
+                    if (blocks.length == 0) {
+                        blocklySearchLabel.innerText = lf("No search results...");
+                    } else {
+                        blocklySearchLabel.innerText = lf("{0} results matching '{1}'", blocks.length, blocklySearchInputField.value.toLowerCase());
```

Comment:
This text is not displayed to the user. It's a hidden label for the screen reader. Maybe a good idea to rename it to "hiddenBlocklySearchLabel" or something like that.